core_impl/pom.xml

```diff
@@ -8,16 +8,32 @@
   <version>0.1.0-SNAPSHOT</version>
 
   <name>Census Java Core Impl</name>
+
+  <description>
+    Census provides a framework to define and collect stats against metrics and
+    to break those stats down across user-defined dimensions.
+  </description>
   <url>https://github.com/google/instrumentation-java</url>
 
+  <packaging>pom</packaging>
+
   <parent>
-    <groupId>com.google.census</groupId>
```

Comment:
Should we create a shared parent that is a child of the google pom but sets the common things for core/core_impl/etc.?